src/loaders/FileLoader.js

```diff
@@ -186,6 +186,21 @@ Object.assign( FileLoader.prototype, {
 
 			if ( request.overrideMimeType ) request.overrideMimeType( this.mimeType !== undefined ? this.mimeType : 'text/plain' );
 
+			if ( this.requestHeader !== undefined ) {
+
+				var keys = Object.keys( this.requestHeader );
```

Comment:
Alright, I'll update to simplify the code.

BTW, about the coding style

> Don't use for in loops on arrays.

Do you allow to use for in on hash if it isn't hot path?